### DISCUSSION TITLE
Z2M update problem

### DISCUSSION THREAD

**Discussion by Fafitude** [Koenkk/zigbee2mqtt#21736] [General]
Good morning,

I would like to update my remote zigbee2mqtt.
I know it's not directly related to jeedom but my Z2M is related to Zigeelinker and people can definitely help me here.
I log in as root and run the following (new) commands:

cd /opt/zigbee2mqtt
./update.sh
Unfortunately I have errors and my Z2M no longer works (fortunately it is a VM and I am restoring). I have attached the errors.
[Z2M.txt](https://github.com/Koenkk/zigbee2mqtt/files/14547722/Z2M.txt)

An idea of ​​the problem ?

Thanks in advance.

**Comment by sjorge** [comment]
As it says in the error message, your node version is too old.

**Comment by Fafitude** [comment]
Thank you for your reply. I'm on debian 10 Buster, what is the command line to update node? Is this possible on this Debian?

**Reply by sjorge** [reply]
Debian usually only updates major versions between debian releases. I guess you could try to upgrade to bookworm. Or look at something like nvm (https://github.com/nvm-sh/nvm) to get a newer node version.

**Comment by Fafitude** [comment]
Thanks, I updated, everything is OK.
